packages/mui-joy/package.json

```diff
@@ -42,7 +42,7 @@
     "@mui/base": "workspace:*",
     "@mui/core-downloads-tracker": "workspace:^",
     "@mui/system": "workspace:^",
-    "@mui/types": "workspace:^",
+    "@mui/types": "workspace:~",
```

Comment:
Updating for consistency but not releasing from this branch, last Joy release was from v5.x, so we need to release this from that branch